pages/education/apply-for-education-benefits/application/1995-STEM.md

```diff
@@ -1,18 +0,0 @@
----
-title: Veteran Request For Change
-heading: Veteran request for change
-entryname: 1995-STEM-edu-benefits
-layout: page-react.html
-description: Use your VA education benefits to pay for college or training programs. Find out which documents you’ll need to apply for benefits, and start your online application today.
-vagovprod: false
-order: 1
-hideFromSidebar: true
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/">Home</a></li>
-    <li><a href="/education/">Education</a></li>
-    <li><a aria-current="page" href="/education/how-to-apply/">Apply for education benefits</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-6e810017caa448956a05d0432d865fb7R5.